worker-document-archetype/documentation/WorkerArchetypeUsage.md

```diff
@@ -20,7 +20,7 @@ The following modules are required to create a Document worker. Each contains it
 **These modules are generated from the worker-document-archetype:**
 - `worker-example` module contains the worker itself and an md documentation explaining the service use of the worker.
 	1. **ExampleWorker** – responsible for doing the work. By default this class implements the BulkDocumentWorker interface. It should be updated to implement the DocumentWorker interface instead if there is no requirement to process documents in batches.
-- `worker-example-container` module is for building the Docker image of the worker and pushing the image to Docker. The module starts a container for RabbitMQ, test-configs, the worker and runs the worker acceptance integration testcases via the `worker-document-testing` module.
+- `worker-example-container` module is for building the Docker image of the worker and pushing the image to Docker. The module starts a container for RabbitMQ, [test-configs](#document-worker-default-configuration), the worker and runs the worker acceptance integration testcases via the `worker-document-testing` module.
```

Comment:
Linking to the default configuration section there doesn't make sense to me.  Also it no longer starts a test container.